### PR TITLE
fix: Fix when minting non divisible token

### PR DESCRIPTION
**Pull request by dzbo**
### Ticket ID

[DEV-2723](https://app.clickup.com/t/2645698/DEV-2723)

### Description

- fix issue when minting non divisible tokens

### CODE REVIEW

**Comment by frozeman** [comment]
Task linked: [DEV-2723 As a User, I can send a LSP7 asset](https://app.clickup.com/t/2645698/DEV-2723)

**Comment by frozeman** [comment]
Task linked: [DEV-2723 As a User, I can send a LSP7 asset](https://app.clickup.com/t/2645698/DEV-2723)

File: src/components/endpoints/Mint.vue [hunk 1]
```diff
@@ -81,18 +81,19 @@ const mint = async () => {
   try {
     switch (tokenType.value) {
       case ContractStandard.LSP7:
-        myToken.value = contract(LSP7Mintable.abi as any, mintToken.value, {
-          gas: DEFAULT_GAS,
-          gasPrice: DEFAULT_GAS_PRICE,
-        })
+        myToken.value = contract(LSP7Mintable.abi as any, mintToken.value)
+
+        debugger
```
Comment (doubleppereira) [inline src/components/endpoints/Mint.vue:86]: debugger?

Comment (dzbo) [reply src/components/endpoints/Mint.vue:86]: That shouldn't go there ;)

**Pull request merged**